Mathlib/Analysis/Meromorphic/FactorizedRational.lean

```diff
@@ -284,3 +374,43 @@ theorem MeromorphicOn.extract_zeros_poles_log {f g : 𝕜 → E} {D : Function.l
   simp only [Pi.smul_apply', Finset.prod_apply, Pi.pow_apply, norm_smul, norm_prod, norm_zpow]
   rw [log_mul (Finset.prod_ne_zero_iff.2 this) (by simp [hg ⟨z, h₃z⟩]), log_prod _ _ this]
   simp [log_zpow]
+
+open Classical in
+/--
+In the setting of `MeromorphicOn.extract_zeros_poles`, compute the trailing
+coefficient of `f` in terms of `divisor f U` and `g x`.
+-/
+theorem MeromorphicOn.meromorphicTrailingCoeffAt_extract_zeros_poles
+    {x : 𝕜} {f g : 𝕜 → E} {D : 𝕜 → ℤ} (hD : D.support.Finite)  (h₁x : x ∈ U) (h₂x : AccPt x (𝓟 U))
+    (hf : MeromorphicAt f x) (h₁g : AnalyticAt 𝕜 g x) (h₂g : g x ≠ 0)
```

Comment:
Can't you remove the `hf` assumption, deducing it from `h` below?

Reply:
@sgouezel I do not think so. For a counterexample, let U be a closed ball, let x be a point on the boundary. Now take f as equal to an analytic function g on the closed ball, and white noise outside of U. Then, all assumptions are satisfied, but f is not meromorphic at x.